terraform/deployments/ecr/main.tf

```diff
@@ -0,0 +1,112 @@
+# This is supposed to be applied in the production environment
+
+terraform {
+  backend "s3" {
+    bucket  = "govuk-terraform-steppingstone-production"
```

Comment:
I'm not sure about putting the terraform state file in the bucket that govuk-aws uses... I worry that could be confusing.

On the other hand, we haven't created a `govuk-terraform-production` bucket yet, because the terraform in this repo hasn't run against production...

🤷🏻  probably not a major problem... I'd be happy to stick with this so long as we've considered it.

Reply:
Yeah I only used it for the sake of expediency as it was already there and we don't really deal with environments yet.

Reply:
🤔 I guess it's fine for now then.